examples/python/ghz.py

```diff
@@ -37,28 +36,9 @@
 for i in range(num_qubits):
     qc.measure(i, i)
 
-###############################################################
-# Set up the API and execute the program.
-###############################################################
-try:
-    IBMQ.load_accounts()
-except:
-    print("""WARNING: No valid IBMQ credentials found on disk. 
-             You must store your credentials using IBMQ.save_account(token, url). 
-             For now, there's only access to local simulator backends...""")
-
 # First version: simulator
```

Comment:
This comment doesn't make sense with line 57 removed

Reply:
Yep, good catch. I removed this in: https://github.com/Qiskit/qiskit-terra/pull/2559/commits/38500c5f0601bfe74430b23decb16f8e96373824